Project.toml

```diff
@@ -1,14 +1,15 @@
 name = "LibCURL"
 uuid = "b27032c2-a3e7-50c8-80cd-2d36dbcbfd21"
-version = "0.5.2"
+version = "0.6.0"
 
 [deps]
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
-Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+LibCURL_jll = "deac9b47-8bc7-5906-a0fe-35ac56dc84c0"
+MozillaCACerts_jll = "14a3606d-f60d-562e-9121-12d972cd8159"
 
 [compat]
-BinaryProvider = "0.5"
-julia = "0.7, 1.0"
+LibCURL_jll = "7.70"
```

Comment:
@aviks is the minimum bump required? I believe we're safe to leave the minimum as 7.68. Reference: https://github.com/JuliaWeb/LibCURL.jl/pull/78#discussion_r419125119

Reply:
@omus I think it's required to pick up HTTP/2 support (`nghttp2`)

Reply:
Having 7.68 here allows installing 7.70 as well

Reply:
Ok, I'll change that back as there's nothing specifically in LibCURL.jl that requires it.

Reply:
Good to know. So if a package depends on LibCURL.jl and wants HTTP/2 support as well, the Package.toml should look like:

```
[deps]
LibCURL = "..."
LibCURL_jll = "..."

[compat]
LibCURL_jll = "7.70"
```
?

Reply:
@geokollias If you want to require at least version 7.70 of `LibCURL_jll` for your package, yes.  Remember that for automerge in General you need upper bounds in compat for all (non-JLL) packages :wink: